invenio_app_ils/cli.py

```diff
@@ -1356,10 +1356,11 @@ def pages():
     is_flag=True,
     help="Skip creating vocabularies."
 )
+@click.option("--with-pages", is_flag=True, help="Create static pages.")
 @click.option("--verbose", is_flag=True, help="Verbose output.")
 @with_appcontext
 def setup(recreate_db, skip_demo_data, skip_file_location, skip_patrons,
-          skip_vocabularies, verbose):
+          skip_vocabularies, with_pages, verbose):
```

Comment:
nit: since everything else works on `skip_` term, we could follow the pattern and do `skip_pages`, and obviously revert the logic.